### ISSUE TITLE
Package request: GameMode

### ISSUE THREAD

**Issue by magnusshtebner** [void-linux/void-packages#19281]
GameMode allows you to change temporarily the CPU priority in favor of the game. And also some other things.

GitHub : https://github.com/FeralInteractive/gamemode/

**Comment by Johnnynator** [comment]
There seems to be still some ties to systemd(?, or something else), `gamemoderun ./game` does not work oob, just spams dbus launch. Need to look closer into it, but Games that set gamemode themself or the example client do work correctly already.

**Comment by kedodrill** [comment]
Any updates on this?

From the 1.6 changelog: "Add option to use elogind"
Guess that allows this to work fine now without systemd? Would love to see this available to install

**Comment by hristoast** [comment]
There are two files that need to be copied into specific places after install:

```
sudo cp $PREFIX/share/dbus-1/services/com.feralinteractive.GameMode.service /usr/share/dbus-1/services/
sudo cp $PREFIX/share/polkit-1/actions/com.feralinteractive.GameMode.policy /usr/share/polkit-1/actions/
```

You probably need to restart dbus/polit; I just rebooted. But after that, I was able to successfully use `gamemoderun`. I think the daemon requires systemd, even when you select elogind.

**Comment by frankfutlg** [comment]
how do i install this?

**Comment by paper42** [comment]
merged in #36564
